OPS/load_stage.sql

```diff
@@ -0,0 +1,275 @@
+/**************************************************************************
+* Copyright 2020 Observational Health Data Sciences and Informatics (OHDSI)
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+* 
+* Authors: Eduard Korchmae, Timur Vakhitov, Dmitry Dymshyts, Christian Reich
+* Date: 2020
+**************************************************************************/
+
+--0. Latest update construction
+DO $_$
+BEGIN
+	PERFORM VOCABULARY_PACK.SetLatestUpdate(
+	pVocabularyName			=> 'OPS',
+	pVocabularyDate			=> TO_DATE ('20200101', 'yyyymmdd'),
+	pVocabularyVersion		=> 'OPS Version 2020',
+	pVocabularyDevSchema	=> 'DEV_OPS'
+);
+END $_$
+;
+--1. Input source tables ops_src_agg and ops_mod_src for all years
+--WbImport example
+/*
+WbImport -file=/home/ekorchmar/Downloads/ops_all/ops2010.csv
```

Comment:
remove this

Reply:
this should go to manual work. we can create another folder where you store all manual work you need

Reply:
This is example of source data import, not manual work